src/main/java/dev/andrylat/vomelianchuk/finances/dialog/CardNumberValidatorDialog.java

```diff
@@ -0,0 +1,29 @@
+package dev.andrylat.vomelianchuk.finances.dialog;
+
+import dev.andrylat.vomelianchuk.finances.consoleutils.CardNumberValidatorConsoleIO;
+import dev.andrylat.vomelianchuk.finances.enums.PaymentSystem;
+import dev.andrylat.vomelianchuk.finances.exceptions.CardValidationException;
+import dev.andrylat.vomelianchuk.finances.exceptions.WrongPaymentSystemException;
+import dev.andrylat.vomelianchuk.finances.card_number_validation.CardNumberValidator;
+
+public class CardNumberValidatorDialog implements Dialog {
+    static final String DESCRIPTION_ACTION = "1 - Card Number Validation";
```

Comment:
Tomorrow you will change ordering in the map. You will go to the every class and change this number?

Reply:
Fixed here: ee85a61